views/editor/nav-options.html

```diff
@@ -77,6 +77,16 @@
                   <img id="sun-white" class="hide" src="/img/icon/theme-sun-white.svg" width="20px" class="hide" height="20px">
                   <img id="sun-green" src="/img/icon/theme-sun-green.svg" width="20px" height="20px">
                 </div>
+              </li>
+              <li>
+                <span>SVG/XML</span>
+                <div title="XML/SVG" class="switch-toggle switch-enabled" id="edit-SVG-toggle">
```

Comment:
Please follow the same HTML structure as the other menu items. You are not using the same div nesting here, and your new code closes off the `<ul>` list, which it should not be doing. 

Also, as above, please make the title a `gettext` string.